docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -4,74 +4,128 @@
-3. Restart Zowe&trade. 
+    Use the following procedure reject encoded slashes.
     
-Requests with encoded slashes are now rejected by the API Mediation Layer.
+    **Follow these steps:**
+        
+    1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
+    2. Find the line that contains the `-Dapiml.service.allowEncodedSlashes=true` parameter and set the value to `false`.
+    3. Restart Zowe&trade. 
+        
+    Requests with encoded slashes are now rejected by the API Mediation Layer.
        
-### apiml.service.corsEnabled
+* **apiml.service.corsEnabled**
 
-By default, CORS are disabled in the API Gateway for the Gateway routes `api/v1/gateway/**`. Allowing CORS for Gateway is necessary to enable CORS at the service level. Use the following procedure to enable CORS.
-    
-**Follow these steps:**
-     
-1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
-2. Find the line that contains the `-Dapiml.service.corsEnabled=false` parameter and set the value to `true`.
-3. Restart Zowe&trade.
-  
-Requests through the Gateway now contain a CORS header. 
+    By default, CORS are disabled in the API Gateway for the Gateway routes `api/v1/gateway/**`. Allowing CORS for Gateway is necessary to enable CORS at the service level. Use the following procedure to enable CORS.
+        
+    **Follow these steps:**
+         
+    1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
+    2. Find the line that contains the `-Dapiml.service.corsEnabled=false` parameter and set the value to `true`.
+    3. Restart Zowe&trade.
+      
+    Requests through the Gateway now contain a CORS header. 
 
-### apiml.security.auth.provider
+* **apiml.service.hostname**
 
-By default, the API Gateway uses z/OSMF as an authentication provider. It is possible to switch to SAF as the authentication
-provider instead of z/OSMF. The intended usage of SAF as an authentication provider is for systems without z/OSMF.
-If SAF is used and the z/OSMF is available on the system, the created tokens are not accepted by z/OSMF. Use
-the following procedure to switch to SAF. 
+    This property is used to set the API Gateway hostname.
 
-**Follow these steps:**
-     
-1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
-2. Find the line that contains the `-Dapiml.security.auth.zosmfServiceId=zosmf` parameter and replace it with `-Dapiml.security.auth.provider=saf`.
-3. Restart Zowe&trade.
+* **apiml.service.port**
+
+    This property is used to set the API Gateway port.
+
+* **apiml.service.discoveryServiceUrls**
+
+    This property specifies the Discovery Service URL used by the service to register to Eureka.
+
+* **apiml.service.preferIpAddress**
+
+    Set the value of this property to `true` if you want to advertise a service IP address instead of its hostname. 
+
+* **apiml.cache.storage.location** 
```

Comment:
I would mention at least that it is necessary for the API ML process to have write access to this location. Though I am not sure whether it is possible to install Zowe in a way where this won't be true. 
@jandadav Do you know what's the current state?

Reply:
I think in Zowe install this would not be an issue, as the parameter is handled. For any customization this would be important.